lib/sms_api.rb

```diff
@@ -19,19 +19,23 @@ def authorization
   def sms_deliver(phone, code)
     return false unless end_point_available?
 
-    response = client.call(:enviar_sms_simples, message: request(phone, code))
+    response = client.call(:sms_text_submit, message: request(phone, code))
     success?(response)
   end
 
   def request(phone, code)
-    { autorizacion:  authorization,
-      destinatarios: { destinatario: phone },
-      texto_mensaje: "Clave para verificarte: #{code}. Gobierno Abierto",
-      solicita_notificacion: "All" }
+    {
+      Authorization: authorization,
+      DeliveryReport: "All",
+      Recipients: { to: phone },
+      Sender: Rails.application.secrets.sms_sender,
+      SMSText: "Clave para verificarte: #{code}. Gobierno Abierto",
```

Comment:
Would it be possible to internationalize this text? So users browsing the web in Galician receive a message written in Galician.

Reply:
Done